src/scripts.js

```diff
@@ -34,90 +33,126 @@ const mainPageView = document.getElementById('mainPageView');
 const recipeResultsView = document.getElementById('recipeResultsView');
 // const recipeResultsStatement = document.getElementById('recipeResultsStatement');
 const recipeInfoView = document.getElementById('recipeInfoView');
-const favoriteRecipesView = document.getElementById('favoriteRecipesView')
 const allRecipesView = document.getElementById('allRecipesView');
+const favoriteRecipesView = document.getElementById('favoriteRecipesView');
+const filteredFavoriteRecipesView = document.getElementById('userFaveRecipeFilteredView');
 
 //RANDOM.
+const recipeTitle = document.getElementById('recipeTitle')
+const currentRecipeImage = document.getElementById('currentRecipeImage')
+const instructionsList = document.getElementById('instructionsList')
+const ingredientsList = document.getElementById('ingredientsList')
+const totalCost = document.getElementById('totalCost')
 // const randomRecipeImage = document.getElementById('randomRecipeImage');
 
-
 //CLASS INSTANSTIATION
 let cookbook = new RecipeBox(recipeData);
 let recipe = new Recipe(recipeData);
+let newUser;
+let currentRecipe; 
+
+// const getRandomIndex = (array) => {
```

Comment:
We need to remember to erase the code comments.

Reply:
Will do!